src/main.js

```diff
@@ -1,84 +1,101 @@
 /* global $ _ Prism Handlebars */
 
-const ROOT_URL = 'https://transportapi.github.io/usage-examples/src/'
+const urlParams = new URLSearchParams(window.location.search)
+const showExperimental =
+  urlParams.has('showExperimental') ? JSON.parse(urlParams.get('showExperimental')) : false
+
+const BRANCH = urlParams.has('branch') ? urlParams.get('branch') : 'master'
```

Comment:
Let's stick for consistent naming for constants. Not sure which is the up to date recommended style, let's choose a style guide and stick to it.